instance.tf

```diff
@@ -9,9 +9,14 @@ resource "aws_instance" "aws_sko_demo" {
   monitoring    = true
   ebs_optimized = true
 
-  associate_public_ip_address = var.data
+  associate_public_ip_address = false
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure public IP address is not used AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances are launched with public IP. This may expose them to the outside world and increase your organisation's attack surface.
 
**Remediation**
AWS EC2 instances are usually used for workload provisioning in the cloud. Therefore, it is recommended not to assign public Ip address or launch them in default VPC. To remediate in Console, launched instances public Ip cannot be disassociated. However, if provisiong using IAC. Ensure in terraform 'aws_instance' resource 'associate_public_ip_address' attribute is not set to true.

source: accurics-devenv